src/screens/HomeScreen.tsx

```diff
@@ -109,14 +113,15 @@ const HomeScreen = ({navigation}: any) => {
     set_is_loading(false);
   }, []);
 
-  console.log(upcoming_movies_list);
+  const handle_selected_movie = (id: number) => {
+    navigation.push('MovieDetail', {
```

Comment:
Use Screens Names from Constants file.
Prefer to write `Stacks`, `Screens`, `Tabs`, and other names/values which are used in multiple files in a constants file .